### PR TITLE
Working example in SimplePlane docs

### PR DESCRIPTION
**Pull request by tomcolaa**
##### Pre-Merge Checklist

- [x] Documentation is changed or added

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
Thanks @tomcolaa!

File: packages/mesh-extras/src/SimplePlane.ts [hunk 1]
```diff
@@ -6,10 +6,14 @@ import { PlaneGeometry } from './geometry/PlaneGeometry';
...
- * };
- * let SimplePlane = new PIXI.SimplePlane(PIXI.Texture.from("snake.png"), points);
+ * let plane = new PIXI.SimplePlane(PIXI.Texture.from("snake.png"));
+ * let geometry = new PIXI.Geometry();
+ *
+ * geometry.addAttribute('positions', [0, 0, 100, 0, 100, 100, 0, 100], 2);
+ * geometry.addAttribute('uvs', [0,0,1,0,1,1,0,1], 2);
+ * geometry.addIndex([0,1,2,1,3,2]);
+ *
+ * plane.geometry = geometry;
```
Comment (bigtimebuddy) [inline packages/mesh-extras/src/SimplePlane.ts]:
```suggestion
 * let plane = new PIXI.SimplePlane(PIXI.Texture.from("snake.png"));
 * let geometry = plane.geometry;
 *
 * geometry.addAttribute('positions', [0, 0, 100, 0, 100, 100, 0, 100], 2);
 * geometry.addAttribute('uvs', [0,0,1,0,1,1,0,1], 2);
 * geometry.addIndex([0,1,2,1,3,2]);
```

Comment (tomcolaa) [reply packages/mesh-extras/src/SimplePlane.ts]: Hi @bigtimebuddy, I tried that as well but it didn't work. That's why I choose to overwrite plane.geometry. I created a playground to test it: https://www.pixiplayground.com/#/edit/0-90FDBNusGDisJw4vIc7

Comment (bigtimebuddy) [reply packages/mesh-extras/src/SimplePlane.ts]: This is probably not a suitable SimplePlane example then. Probably a better example for Mesh.

**Comment by ivanpopelyshev** [comment]
Exca made demo: https://pixijs.io/examples/#/demos-basic/simpleplane.js we can just take it from there

**Comment by bigtimebuddy** [comment]
I'm going to close this, this doc is not the right approach. I'd suggest something much simpler or removing the SimplePlane example all together from the code. The example Ivan pointed to is a much better example how to use the class.

**Pull request closed without merging**